### ISSUE TITLE
Incorrect plot command crashes spyder

### ISSUE THREAD

**Issue by spyder-issues-migrator** [spyder-ide/spyder#491]
_From [cassioma...@gmail.com](https://code.google.com/u/111021380268968272998/) on 2010-12-06T11:39:27Z_

What steps will reproduce the problem?

 1. Open spyder
2. Type the following code in the temp file:

from pylab import *

x = [1, 2]
y = [3, 4]

plot(x,y, c = '[0.5]')
show()

3. Run with F5.

What is the expected output? 
Stack trace, showing that '[0.5]' is an invalid color...

What do you see instead? Spyder crashes. What version of the product are you using? On what operating system? Spyder 2.0.3 on Fedora 14. 

Please provide any additional information below

.

_Original issue: http://code.google.com/p/spyderlib/issues/detail?id=491_

**Comment by spyder-issues-migrator** [comment]
_From [ccordoba12](https://code.google.com/u/ccordoba12/) on 2010-12-15T08:05:51Z_

I'm sorry but I can't reproduce your bug. I'm getting the right stack trace and spyder doesn't crashes.

Please activate the "Internal console" plug-in and try to check what messages appear there before spyder crashes.

Also if you have spyder version 1.0 installed, please uninstall it and reinstall version 2.0 again. That could be the cause of your problem.

Carlos

**Comment by spyder-issues-migrator** [comment]
_From [cassioma...@gmail.com](https://code.google.com/u/111021380268968272998/) on 2010-12-15T08:11:10Z_

Carlos,

This is what I get starting spyder with the -d flag and then running the script:

Start of MainWindow constructor
Initializing...
End of MainWindow constructor
*** Start of MainWindow setup ***
Loading object inspector...
Loading editor...

Loading file explorer...
Loading history plugin...
Loading online help...
Loading project explorer...
Loading external console...
Loading namespace browser...
Setting up main window...
Setting up window...
*** End of MainWindow setup ***
Segmentation fault (core dumped)


Im using Fedora ( https://fedoraproject.org/ ). It comes with SELinux enabled. Perhaps it is some kind of related issue?

**Comment by spyder-issues-migrator** [comment]
_From [ccordoba12](https://code.google.com/u/ccordoba12/) on 2010-12-15T08:19:48Z_

I'm using Kubuntu 10.10 so, as you say, probably SELinux is your problem. Can you deactivate it and try again?

Please don't forget to use the internal console and check the (possible) error messages that could appear there.

**Comment by spyder-issues-migrator** [comment]
_From [cassioma...@gmail.com](https://code.google.com/u/111021380268968272998/) on 2010-12-15T08:22:09Z_

Carlos,

There is not enough time to check the internal console. The application crashes and closes when the segfault occurrs. Is the information logged somewhere else? I'll try later with selinux disabled to see if it crashes.
Thanks.

**Comment by spyder-issues-migrator** [comment]
_From [ccordoba12](https://code.google.com/u/ccordoba12/) on 2010-12-15T12:03:43Z_

Cassio, can you see at least something printed on the internal console? You can hit F5 while you are viewing it, and the file still will be run on the ipython console. If not, I would say you have a problem with your PyQt installation.

**Comment by spyder-issues-migrator** [comment]
_From [cassioma...@gmail.com](https://code.google.com/u/111021380268968272998/) on 2010-12-22T08:06:56Z_

Carlos,

I have tried running it with selinux disabled and it still crashes. Hitting F5 makes the window turn to the ipython console, instead of the internal console. After it runs, the application segfaults and exits. By the way, I tried it now with version 2.0.5 which made its way into the Fedora repos.
Thanks.
